### ISSUE TITLE
deadline exceeded should returns 1 instead of 0

### ISSUE THREAD

**Issue by quantonganh** [golangci/golangci-lint#41]
```
...
INFO[0082] processing took 667.366µs with stages: max_same_issues: 664.846µs, max_from_linter: 1.138µs, path_prettifier: 394ns, cgo: 227ns, exclude: 203ns, diff: 181ns, nolint: 173ns, max_per_file_from_linter: 154ns, uniq_by_line: 50ns
WARN[0082] 0/9 linters finished: deadline exceeded: try increase it by passing --deadline option
Congrats! No issues were found.
INFO[0082] Extracting issued lines took 0s

❯❯❯❯ echo $?
0
```

**Comment by golangci** [comment]
@quantonganh thank you!
fixed in https://github.com/golangci/golangci-lint/pull/44, check in master, please

**Comment by quantonganh** [comment]
```
WARN[0076] 0/9 linters finished: deadline exceeded: try increase it by passing --deadline option
Congrats! No issues were found.
INFO[0076] Workers idle times: #1: 84.316µs, #3: 72.969µs, #4: 92.428µs
INFO[0076] Extracting issued lines took 0s
❯❯❯❯ echo $?
4
```

Exit code is ok now but IMO, "Congrats! No issues were found." should not be printed.

**Comment by golangci** [comment]
fixed "Congrats! No issues were found." in master
